conda/recipes/ucxx/conda_build_config.yaml

```diff
@@ -34,4 +34,4 @@ gtest:
   - ">=1.13.0"
 
 rapids_version:
-  - "25.6.*"
+  - ">=25.6,<25.8"
```

Comment:
Is this necessary? If so, https://github.com/rapidsai/ucxx/blob/branch-0.44/ci/release/update-version.sh#L44 needs to be updated, but the script doesn't track the next version currently, so it would need to calculate it.

Also, if so, I think it might need to be `<25.8.0a0`.

Reply:
I think this change is not necessary.

Reply:
No, I got hung up on some channel priority conflicts but it was due to a local package I had sitting in my build output.  Reverted.